### ISSUE TITLE
Here's an idea please implement i would do it but I'm still learning how to code!!!

### ISSUE THREAD

**Issue by Mr-Myabi** [harmsk/atom-html-preview#225]
Hey you sexy beasts, pretty pretty please implement a Pesticides like feature in HTML preview. Not only will I be able to see the preview of my site but I ill see the containers too.

**Comment by mondele** [comment]
If you really want this implemented, you should edit the title of the issue to show what you're asking. Then, give more detail in the comment.

**Comment by Mr-Myabi** [comment]
> 

my bad, man, I'm new to the tech scene and am learning how yall communicate. Let me try again, thank you for the advice 💙

**Comment by Mr-Myabi** [comment]
Did I get closer to what you wanted? 👉👈
